community-plugins.json

```diff
@@ -5788,5 +5788,13 @@
         "author": "Ryan Gomba",
         "repo": "ryangomba/obsidian-todo-sort",
         "branch": "main"
+    },
+    {
+        "id": "obsidian-console",
+        "name": "Console Markdown Plugin",
```

Comment:
You might want to choose a simpler name, it's a bit redundant to include the word "Plugin" in the plugin name.